exercises/G-conditionals-2/exercise-3.js

```diff
@@ -8,7 +8,16 @@
   - Otherwise the grade is "F"
 */
 
-function calculateGrade(mark) {}
+function calculateGrade(mark) {
+  if (mark >= 80) {
+    return `A`;
+  } else if (mark < 80 && mark > 60) {
+    return `B`;
+  } else if (mark <= 60 && mark >= 50) {
+    return `C`;
+  }
+  return `F`;
+}
```

Comment:
Your work here is definitely correct, but you can make it shorter by removing some code, for example you can remove (mark < 80 && ) from second if, as the compiler will come here only when mark < 80. your code can be:
if(mark >= 80){
return "A";
} else if (mark > 60) {
return "B";
} else if (mark >50) {

Reply:
Thank you for your feedback Ahmad! It has been very helpful :)